### PR TITLE
 Bug 1175134 - Present the settings in a modal view on iPad

### PR DESCRIPTION
**Pull request by BigxMac**
The iPad Settings app now modals in the form of a sheet.

https://bugzilla.mozilla.org/show_bug.cgi?id=1175134

### CODE REVIEW

**Comment by st3fan** [comment]
Screenshots for @darrinhenein 

* https://www.dropbox.com/s/y9lfnwb58r3ib1j/Screenshot%202015-06-16%2013.10.17.png?dl=0
* https://www.dropbox.com/s/3nvadwxyzexx3yf/Screenshot%202015-06-16%2013.10.39.png?dl=0

**Comment by st3fan** [comment]
@BigxMac This looks good. Can you also present the settings in a similar way when you hit the 'Sign In to Firefox' button on the Intro page? From there it opens the settings in full screen.

File: Client/Frontend/Intro/IntroViewController.swift [hunk 1]
```diff
@@ -264,7 +264,7 @@ class IntroViewController: UIViewController, UIScrollViewDelegate {
     }
 
     func SELlogin() {
-        delegate?.introViewControllerDidRequestToLogin(self)
+		delegate?.introViewControllerDidRequestToLogin(self)
```
Comment (BigxMac) [inline Client/Frontend/Intro/IntroViewController.swift]: Not sure why this was commited

**Comment by darrinhenein** [comment]
LGTM, thanks!

**Pull request merged**